src/server/utils/postDeploy.js

```diff
@@ -0,0 +1,18 @@
+import getRethink from 'server/database/rethinkDriver';
+
+const flushSocketConnections = async () => {
+  const r = getRethink();
+  return r.table('User')
+    .update({
```

Comment:
Does this map over every single user in the table?

Reply:
yep! when the server restarts there's no graceful shutdown where the socket.disconnect is guaranteed to be called